pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -194,8 +195,8 @@ func (r *reconcilerBase) createOrPatchDeployment(ctx context.Context, declared *
 		return controllerutil.OperationResultCreated, nil
 	}
 
-	// TODO: check if VPA is enabled
-	vpaEnabled := false
+	//check if allow vertical scale is enabled
+	allowVerticalScale := r.allowVerticalScale
```

Comment:
You can just use `r.allowVerticalScale` directly

Reply:
used r.allowVerticalScale directly